roles/container_runtime/tasks/package_docker.yml

```diff
@@ -31,6 +31,20 @@
   register: result
   until: result is succeeded
 
+ - name: Install cri-tools
+   package:
+    name: "cri-tools"
+    state: present
+
+- name: Configure cri-tools for dockershim
+  copy:
+    contents: |
+      runtime-endpoint: unix:///var/run/dockershim.sock
+      image-endpoint: unix:///var/run/dockershim.sock
+    dest: /etc/crictl.yaml
+  when:
+  - not (openshift_use_crio | bool)
```

Comment:
@sdodson I am not sure whether this is the right way. We want this to be installed on masters even when nodes are expected to have crio installed.

Reply:
Do you have suggestions?

Reply:
That will work as long as that's all that you'd want in the file. If the user may add other values that are unrelated we should use http://docs.ansible.com/ansible/lineinfile_module.html

Reply:
okay! yeah, that's what we would want in that file as long as crio isn't installed as well. So this is for use cases where we run docker on the masters and crio on the nodes for example.

Reply:
sounds good